src/EditorFeatures/Core/SettingsData/Analyzers/AnalyzerSettingInfo.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.Editor.SettingsData
```

Comment:
nit: Consider matching the namespace with directory structure, i.e. `Microsoft.CodeAnalysis.Editor.SettingsData.Analyzers`